test/tests/kernels/ode/parsedode_pp_test.i

```diff
@@ -0,0 +1,74 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 0
+  xmax = 1
+  ymin = 0
+  ymax = 1
+  nx = 2
```

Comment:
I see, only two elements. Does it do something bad if you scale beyond 2 processors? Just curious.

Reply:
Turns out this works. It was just copy paste from the test above.